tests/cue_test.c

```diff
@@ -0,0 +1,74 @@
+
+#include "sfconfig.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <sndfile.h>
+
+
+static int test_cues (const char *filename);
+int test_cues (const char *filename)
+{
+    SNDFILE    *file;
+    SF_INFO	sfinfo;
+
+    int i, err;
+    uint32_t count = 0;
+    SF_CUES_VAR(0) *info;
+
+    if ((file = sf_open(filename, SFM_READ, &sfinfo)) == NULL)
+    {
+	printf("can't open file '%s'\n", filename);
+	return 0;
+    }
+
+    printf("\n---- get cues of file '%s'\n", filename);
+
```

Comment:
This test program's output to stdout should follow the conventions of the other test programs.